com.unity.render-pipelines.high-definition/Documentation~/Density-Volume.md

```diff
@@ -34,16 +34,15 @@ To create a Density Volume, right click in the Hierarchy and select __Rendering
 
 ## Creating a Density Mask Texture
 
-1. In image-editing software of your choice, prepare a grayscale 2D texture with values between 0 and 1 and of size 1024x32. This size describes a 3D texture of size 32x32x32 with 32 slices laid out one after another.
-2. Import this texture into the Unity Editor and set the Texture Import Settings:
+1. In image-editing software of your choice, prepare a grayscale flipbook texture and [import it as a 3D texture](https://docs.unity3d.com/2020.2/Documentation/Manual/class-Texture3D.html). For example, a texture of size 1024x32 describes a 3D texture of size 32x32x32 with 32 slices laid out one after another.
```

Comment:
@adrien-de-tocqueville can you also update the upgrade guide 2020.1 to 2020.2 to explain that we have remove this tool and we should use the regular importer? Thanks

Reply:
The rest of the doc looks good. @adrien-de-tocqueville, can you ping me when you've added the entry to the upgrade guide so I can have a look at that?

Reply:
@JordanL8 done